releasenotes/notes/1.3/add-mcx-plugins-85e5b248692a36db.yaml

```diff
@@ -17,12 +17,12 @@ features_transpiler:
     * :class:`.MCXSynthesisNoAuxV24`, based on :func:`.synth_mcx_noaux_v24`.
     * :class:`.MCXSynthesisGrayCode`, based on :func:`.synth_mcx_gray_code`.
 
-    As well:
+    As well as:
 
-    * :class:`.MCXSynthesisDefault`, choosing the most efficient synthesis
+    * :class:`.MCXSynthesisDefault`, for choosing the most efficient synthesis
       method based on the number of clean and dirty ancilla qubits available.
 
-    As an example, consider how the transpilation of the following circuit::
+    As an example, consider the transpilation of the following circuit::
```

Comment:
This might be a better choice to use the plot directive and change the last line of the code example to `transpile(qc).draw("mpl")`